src/App.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  BrowserRouter, Routes, Route,
+} from 'react-router-dom';
+
+import { Container } from '@mui/material';
+import ResponsiveDrawer from './components/ResponsiveDrawer';
+import Details from './pages/Details';
+import Rooms from './pages/Rooms';
+import AddReservation from './components/AddReservation';
+import Login from './pages/Login';
+import SignUp from './pages/SignUp';
+import CreateRoom from './pages/CreateRoom';
+import PrivateRoute from './components/PrivateRoute';
+import MyReservations from './components/MyReservations';
+import { displayRooms } from './redux/actionCreator';
+import DeleteRoom from './pages/DeleteRoom';
+import ReserveRoom from './components/ReserveRoom';
+
+function App() {
```

Comment:
- [ ] You are using the `function` keyword for many of your functions.  Kindly refactor them to ES6 arrow functions.

## Explanation

In the industry, the `function` keyword has been soft deprecated almost everywhere in favor of ES6.  On your next job, you'll be most likely required to write ES6.

## Tips and extra help

Just perform a search on Vscode for the `function` word.  Refactor to arrow function as you find them (will take you about 5-10 minutes at most, don't worry).

That way, your project will comply with the latest coding standards 💪